config/routes.rb

```diff
@@ -2,6 +2,12 @@
   get 'postals/:id', to: "postals#show"
   resources :interest_places
   resources :complaints
+	resources :adminpanel
```

Comment:
why is necessary the indentation?

Reply:
I don't know why it shows indentation, if it does not have